server/state-cache/index.js

```diff
@@ -1,12 +1,15 @@
 /**
  * External Dependencies
+ *
+ * @format
  */
+
```

Comment:
I think those blank lines are obsolete after the comment block.

Reply:
right now the desired format was actually to add in that newlines 🤷‍♂️.  I'm not passionate about this

Reply:
I see... okey